app/views/govuk_publishing_components/components/contextual_sidebar/_ukraine_cta.html.erb

```diff
@@ -7,8 +7,16 @@
 <%= tag.div class: "gem-c-contextual-sidebar__cta gem-c-contextual-sidebar__cta--ukraine", data: { module: "gem-track-click" } do %>
```

Comment:
I think you need to initialise the `ga4-link-tracker` module here.

Reply:
D'oh! Thanks, well spotted. I've now added this, as well as a test for this functionality (thanks to @Rosa-Fox for the help!). See the middle commit for changes.